src/scene/webscene/index.jsx

```diff
@@ -101,7 +104,7 @@ class Webscene extends Component {
         );
         // assign only fist ground layer to avoid out-of-sync layer settings, e.g. visibility
         // as there is no way (yet) to add a group layer to the ground of a SceneView
-        groundLayer = filteredGroundLayers[0];
+        groundLayer = filteredGroundLayers ? filteredGroundLayers[0] : null;
```

Comment:
How can `filteredGroundLayers` be falsy?

Reply:
In that case, I would rather keep groundLayer: undefined and handle it downstream (I think it already is).

```suggestion
        groundLayer = filteredGroundLayers[0];
```

Reply:
Yes, that's what I meant...